lib/go-tc/profiles.go

```diff
@@ -91,6 +92,28 @@ type Profile struct {
 	Parameters      []ParameterNullable `json:"params,omitempty"`
 }
 
+// ProfilesResponseV5 is a list of profiles returned by GET requests.
+type ProfilesResponseV5 struct {
+	Response []ProfileV5 `json:"response"`
+	Alerts
+}
+
+// A ProfileV5 represents a set of configuration for a server or Delivery Service
+// which may be reused to allow sharing configuration across the objects to
+// which it is assigned. Note: Field LastUpdated represents RFC3339
+type ProfileV5 struct {
+	ID              int                 `json:"id" db:"id"`
+	LastUpdated     time.Time           `json:"lastUpdated" db:"last_updated"`
+	Name            string              `json:"name" db:"name"`
+	Parameter       string              `json:"param"`
```

Comment:
I just checked, and removing this field causes no functional issues. No unit tests failed, no integration tests failed, and no API responses changed as far as I could tell. It looks totally unused.